src/pages/Books.js

```diff
@@ -0,0 +1,16 @@
+import BookContainer from '../components/Bookstore';
+import Header from '../components/Header';
+import InputBook from '../components/InputeBook';
+
+const Books = () => (
+  <div>
+    <Header />
+    <div style={{ padding: '10px 50px' }}>
+      <BookContainer />
```

Comment:
- [ ] Your **`BookContainer`** should take in book data as a prop and render its `title` and `author` (or category here). Kindly fix this.
- [ ] It should also contain a placeholder **Remove** button for removing books. Kindly add it

Reply:
please read the Microverse description [here](https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/project_init_project_wth_comps.md)

Reply:
@vic778 I have read it. Please, do not hesitate to highlight what is unclear. As stated in the requirements, the **_building blocks_** of your app should be re-usable components (renders dynamic UI based on prop data) and books should be listed.

There should be a reusable component representing a single book and that component should take in book data as props. And it should have a Remove button (no implementation yet). Just a placeholder button. You have no such button.